docs/DeveloperGuide.md

```diff
@@ -1,34 +1,1135 @@
+   2. If `JDK 11` is listed in the drop down, select it. Otherwise, click `New…` and select the directory where you installed `JDK 11`.
+   3. Click `OK`.
+4. Click `Import Project`.
+5. Locate the `build.gradle` file and select it. Click `OK`.
+6. Click `Open as Project`.
+7. Click `OK` to accept the default settings if prompted. 
 
-## User Stories
+[&#8593; Return to Top](#developer-guide)
+
+## 3. <a id="design">Design</a>
+This section provides a high level overview of our application, The Schwarzenegger.
+### 3.1. <a id="architect">Architecture</a>
+
+![Architecture](pictures/zesong/Architecture.png)
+
+The image above explains the design of the application, The Schwarzenegger. 
+
+The main driver of the application is `Main: Duke`. It is responsible for mainly two phases:
+- At application launch
+    - This class will initialise the components in the correct sequence and is in charge of connecting them with each other.
+- At shut down
+    - This class will invoke cleanup method for the components when necessary.
+    
+In addition to that, the architecture of The Schwarzenegger is broken down into several packages, mainly the following: 
+- `Ui`: This class mainly handles the interactions with user of the application.
+- `Parser`: This class mainly handles the parsing and handling of user commands.
+- `Command`: This class handles the type of command.
+- `Profile`: This class manages the data of the user.
+- `Diet`: This class manages the diet recording sessions.
+- `Workout`: This class manages the data workout recording sessions.
+- `Storage`: This class reads data from and writes data back into a text file for future uses.
+
+[&#8593; Return to Top](#developer-guide)
+
+### 3.2. <a id="ui-component">Ui Component</a>
+![Ui Component](images/UI_Component.png)
+
+API: `Ui.java`
+ 
+The `Ui` package is a combination class where all interaction will be made through this component
+ 
+The `Ui` component,
+
+* Takes in user input
+* Prints out response messages
+
+[&#8593; Return to Top](#developer-guide)
+
+### 3.3. <a id="logic-component">Logic Component</a>
+
+![Logic Component](images/logic_update.png)
+
+1. `The Schwarzenegger` uses `Parser` classes to parse the user command.
+2. This splits the user input into interpretable portions by other functions.
+
+[&#8593; Return to Top](#developer-guide)
+
+### 3.4. <a id="model-component">Model Component</a>
+
+![Model Component](pictures/zesong/model.png)
+
+The Model component contains `Profile`, `DietManager`,
+`PastRecord` and `WorkoutSession` classes.
+* Profile: Stores the user profile data.
+* DietManager: Stores all past diet records.
+* PastRecord: Stores meta information of each WorkoutSession files.
+* WorkoutSession: Stores the exercise data done in each workout session.
+
+[&#8593; Return to Top](#developer-guide)
+
+### 3.5. <a id="storage-component">Storage Component</a>
+
+![Storage Class Diagram](images/Storage.png)
```

Comment:
I can't view this picture on my side, it has the same issue as the ones described above